frontend/src/app/modules/registration/registration-form/registration-form.component.html

```diff
@@ -64,51 +64,102 @@
-                <small *ngIf="formGroup.controls.organizationSlug.value?.length > 50">Max length 50</small>
-                <small *ngIf="formGroup.controls.organizationSlug.errors?.notUnique">Slug is not unique</small>
-                <small *ngIf="formGroup.controls.organizationSlug.hasError('pattern')">May contain letters, numbers,
-                    space, dashes and underscores</small>
+                <small *ngIf="personalDetail.controls.confirmPassword.errors.required">Сan not be empty</small>
+                <small *ngIf="personalDetail.controls.confirmPassword.errors.notEqual">Passwords must match</small>
             </div>
         </div>
     </div>
     <div class="submit-block">
         <span>
-            <button pButton class="submit-button p-button-outlined" label="Submit" (click)="onSubmit()"
-                [disabled]="!formGroup.valid"></button>
+            <button pButton class="submit-button p-button-outlined" label="Next" (click)="next()"
+                [disabled]="!personalDetail.valid"></button>
         </span>
     </div>
 </form>
+<div *ngIf="step === stepRegistration.Second">
+    <p-tabView (onChange)="handleChange($event)">
+        <p-tabPanel >
+            <ng-template pTemplate="header">
+                <div class="header">
+                    <span>Create new organization</span>
+                </div>
+            </ng-template>
+            <form [formGroup]="ogranizationDetail" action="/signup" class="reg-form organization-form" method="POST"
+                *ngIf="indexOfSelectedTab === tabRegistration.CreateOrganization">
+                <div class="reg-block">
+                    <div class="input-block">
+                        <span class="p-float-label">
+                            <input id="float-input-comp-name" type="text" formControlName="organizationName" name="comp-name"
+                                pInputText [(ngModel)]="organization.name">
+                            <label for="float-input-comp-name">Organization name</label>
+                        </span>
+                        <div *ngIf="organization_step && ogranizationDetail.controls.organizationName.touched && ogranizationDetail.controls.organizationName.invalid"
+                            class="validation">
+                            <small *ngIf="ogranizationDetail.controls.organizationName.errors.required">Сan not be empty</small>
+                            <small *ngIf="ogranizationDetail.controls.organizationName.value?.length < 3">Min length 3</small>
+                            <small *ngIf="ogranizationDetail.controls.organizationName.value?.length > 50">Max length 50</small>
+                            <small *ngIf="ogranizationDetail.controls.organizationName.hasError('pattern')">May contain letters, numbers,
+                                space, special character: !#$%&‘*+—/=?^_`&#123;|&#125;~</small>
```

Comment:
Please delete list of special characters

Reply:
Fix